docs/user-guide/src/version_support.md

```diff
@@ -5,26 +5,29 @@ The Cluster API Provider Metal3 (CAPM3) team maintains the following branches fo
 - CAPM3
     - main
         - v1beta1
-    - release-0.5
-        - v1alpha5
+    - release-1.2
+        - v1beta1
+    - release-1.1
+        - v1beta1
 
-Currently, in Metal³ organization only CAPM3 and IPAM follow CAPI release cycles. The supported versions for CAPM3 and
-IPAM releases are as follows:
+Currently, in Metal³ organization only CAPM3 and IPAM follow CAPI release cycles. The supported versions (excluding release candidates) for CAPM3 and IPAM releases are as follows:
 
 - CAPM3
     - v1beta1
-        - v1.1.2, v1.1.1, v1.1.0
+        - v1.2.0, v1.1.3, v1.1.2, v1.1.1, v1.1.0
     - v1alpha5
         - v0.5.5, v0.5.4, v0.5.3, v0.5.2, v0.5.1, v0.5.0
 - IPAM
     - v1alpha1
-        - v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
+        - v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
 
 The compatability of IPAM and CAPM3 API versions with CAPI is discussed [here](https://github.com/metal3-io/ip-address-manager#compatibility-with-cluster-api).
 
-Since BMO and Ironic do not follow similar release cycles they are backward compatible. However, we always tag the BMO and Ironic code base whenever we do a release in CAPM3. The tags have a prefix **capm3-** and the suffix is always the corresponding capm3- release version. So for example if we cut a `v1.0.0` release for CAPM3 we create a tag in the BMO and Ironic code base with `capm3-v1.0.0`. Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
+Since BMO and Ironic do not follow similar release cycles they are backward compatible. However, we always tag the BMO and Ironic code base whenever we do a release in CAPM3. The tags have a prefix **capm3-** and the suffix is always the corresponding capm3- release version. So for example, if we cut a `v1.0.0` release for CAPM3 we create a tag in the BMO and Ironic code base with `capm3-v1.0.0`. Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
```

Comment:
```suggestion
Since BMO and Ironic do not follow similar release cycles they are backward compatible. However, we used to tag the BMO and Ironic code base whenever we do a release in CAPM3 prior to CAPM3 release 1.2.0. The tags used to have a prefix **capm3-** and the suffix was always the corresponding capm3- release version. So for example, if we cut a `v1.0.0` release for CAPM3 we created a tag in the BMO and Ironic code base with `capm3-v1.0.0`. Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
```

Reply:
Actually I noticed it should be: `prior to CAPM3 release 1.1.3 ` because if we note 1.2.0 it might be confusing since 1.1.3 also does not have bmo tagged